syntax = "proto3";

import "google/protobuf/wrappers.proto";
import "google/protobuf/timestamp.proto";

option csharp_namespace = "KgdIndividualService";

package kgd_individual_service;

service KgdIndividual{
  rpc LoadData(LoadDataRequest) returns (LoadDataReply);
}

message LoadDataRequest{
  int64 iin = 1;
}

message LoadDataReply {
  string message = 1;
  Data data = 2;
}

message Data {
  google.protobuf.StringValue first_name = 1;
  google.protobuf.StringValue last_name = 2;
  google.protobuf.StringValue middle_name = 3;
  google.protobuf.Timestamp begin_date = 4;
}
